tests/string_test.py

```diff
@@ -562,6 +566,13 @@ def test_case_change(self):
         expected = ak.arange(30) >= 20
         self.assertListEqual(isupper.to_ndarray().tolist(), expected.to_ndarray().tolist())
 
+        # first 10 all lower, middle 10 mixed case (not lower or upper), last 10 all title
+        t = ak.concatenate([lower, s, title])
+
+        istitle = t.is_title()
+        expected = ak.arange(30) >= 20
+        self.assertListEqual(istitle.to_ndarray().tolist(), expected.to_ndarray().tolist())
+
```

Comment:
I think the better way to do this is to modify the existing `lu` to something like this
```python
# first 10 all lower, second 10 mixed case (not lower, upper, or title), third 10 all upper, last 10 all title
lmut = ak.concatenate([lower, s, upper, title])

islower = lmut.is_lower()
expected = 10 > ak.arange(40)
self.assertListEqual(islower.to_ndarray().tolist(), expected.to_ndarray().tolist())

isupper = lmut.is_upper()
expected = (30 > ak.arange(40)) & (ak.arange(40) >= 20)
self.assertListEqual(isupper.to_ndarray().tolist(), expected.to_ndarray().tolist())

istitle = lmut.is_title()
expected = ak.arange(40) >= 30
self.assertListEqual(istitle.to_ndarray().tolist(), expected.to_ndarray().tolist())
```

Reply:
I know i wrote the original, so this one is on me. But I would also rename `s` to `mixed` so it's more clear